PerimeterXModule/Internals/CredentialsIntelligence/PxLoginData.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Web;
+using Jil;
+using PerimeterX.CustomBehavior;
+
+namespace PerimeterX
+{
+    public class PxLoginData
```

Comment:
When I see "data", I think of data fields, not methods. I think maybe a better name for this class would be `CredentialIntelligenceManager` or something like that. What do you think?